cmd/jaeger/config-tail-sampling-service-name-policy.yaml

```diff
@@ -0,0 +1,42 @@
+service:
+  extensions: [jaeger_storage, jaeger_query]
+  pipelines:
+    traces:
+      receivers: [otlp]
+      processors: [tail_sampling]
+      exporters: [jaeger_storage_exporter]
+  telemetry:
+    logs:
+      level: DEBUG
+
+extensions:
+  jaeger_query:
+    trace_storage: some_storage
+  jaeger_storage:
+    backends:
+      some_storage:
+        memory:
+          max_traces: 100000
+
+receivers:
+  otlp:
+    protocols:
+      grpc:
+      http:
+        endpoint: "0.0.0.0:4318"
+
+processors:
+  tail_sampling:
+    decision_wait: 5s
+    policies:
+      [
+        {
+          name: filter-by-attribute,
+          type: string_attribute,
+          string_attribute: { key: service.name, values: [tracegen-00, tracegen-03] }
+        },
+      ]
```

Comment:
this is an odd syntax for YAML, how does it even parse? Don't you need `policies: |` ?

Reply:
I was following the [README for the tailsamplingprocessor](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/processor/tailsamplingprocessor/README.md) and this seems to be how they've done it there.